run_tests.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * This script runs tests for all packages in `./packages` and
+ * example projects in `./examples`.
+ */
+
+const fs = require('graceful-fs');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const PACKAGES_DIR = './packages';
+const EXAMPLES_DIR = './examples';
+
+const packages = fs.readdirSync(PACKAGES_DIR)
+  .map((file) => path.resolve(PACKAGES_DIR, file))
+  .filter((f) => fs.lstatSync(path.resolve(f)).isDirectory());
+
+const examples = fs.readdirSync(EXAMPLES_DIR)
+  .map((file) => path.resolve(EXAMPLES_DIR, file))
+  .filter((f) => fs.lstatSync(path.resolve(f)).isDirectory());
+
+
+packages.forEach((dir) => {
+  console.log('Testing package: ' + dir);
+  execSync('npm test', {cwd: dir, stdio: [0, 1, 2]});
+});
+
+examples.forEach((dir) => {
+  console.log('Testing example: ' + dir);
+  execSync(
+    'npm i && npm test', {cwd: dir, stdio: [0, 1, 2]}
```

Comment:
nit: I recommend one of these code styles:

```js
execSync('npm i && npm test', {
  cwd,
  stdio: [0, 1, 2],
});

// or
execSync(
  'npm i && npm test',
  {
    cwd,
    stdio: [0, 1, 2],
  }
);
```

The first one is ok when the last argument is the only object expression :)